### ISSUE TITLE
[MultiSelect] NVDA/JAWS disable arrow keys navigation shortcuts

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#4261]
**Describe the bug**
Using NVDA or JAWS disables the built-in arrow key navigation of the selected chips. 
By default, the user can navigate through the selected chips and remove the desired ones if necessary. 
https://www.telerik.com/kendo-angular-ui/components/dropdowns/multiselect/keyboard-navigation/

But when using a screen reader, the arrow key shortcuts are not working, and cannot focus previous or next chip.

Reported in ticket: **1646620**

**To Reproduce**
1. Go to https://stackblitz.com/edit/angular-fgws7i?file=src%2Fapp%2Fapp.component.ts
2. Navigate through the selected options with arrow keys. 
3. Turn on NVDA
4. Navigate through the selected options with arrow keys. Now the keys aren't working.

**Comment by stnikolova** [comment]
The fix is available in version **16.2.0**.
